go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -385,17 +386,6 @@ func (tsv *TabletServer) IsHealthy() error {
 	return nil
 }
 
-// IsServingType returns true if the tablet type is one that should be serving to be healthy, or false if the tablet type
-// should not be serving in it's healthy state.
-func IsServingType(tabletType topodatapb.TabletType) bool {
```

Comment:
This particular function feels like it belongs better in `topoproto` than in a subpackage of `vttablet/tabletserver`

Reply:
Just moved this into topoproto itself.